lib/crossdoc/tree.rb

```diff
@@ -16,7 +16,7 @@ def initialize(attrs={})
 
     attr_reader :io, :is_svg
 
-    def download
+    def download(num_images)
```

Comment:
Rather than passing num_images here, I would prefer using a boolean parameter like `skip_processing: false` and then in download_images pass in `@doc.images.count < 6`